data/data-pipeline/data_pipeline/tests/test_score_calculation.py

```diff
@@ -1,302 +1,152 @@
-def env_percentile_threshold():
-    return ScoreNarwhal(final_score_df).ENVIRONMENTAL_BURDEN_THRESHOLD
-
-
-# Climate factor thresholds
-@pytest.fixture
-def expected_pop_loss_percentile_columns():
-    return [
-        field_names.EXPECTED_POPULATION_LOSS_EXCEEDS_PCTILE_THRESHOLD,
-        field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD
-        + field_names.PERCENTILE_FIELD_SUFFIX,
-    ]
-
-
-@pytest.fixture
-def expected_ag_loss_percentile_columns():
-    return [
-        field_names.EXPECTED_AGRICULTURAL_LOSS_EXCEEDS_PCTILE_THRESHOLD,
-        field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD
-        + field_names.PERCENTILE_FIELD_SUFFIX,
-    ]
-
-
-@pytest.fixture
-def expected_bldg_loss_percentile_columns():
-    return [
-        field_names.EXPECTED_BUILDING_LOSS_EXCEEDS_PCTILE_THRESHOLD,
-        field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD
-        + field_names.PERCENTILE_FIELD_SUFFIX,
-    ]
-
-
-@pytest.fixture
-def expected_flood_percentile_columns():
-    return [
-        field_names.HIGH_FUTURE_FLOOD_RISK_FIELD,
-        field_names.FUTURE_FLOOD_RISK_FIELD
-        + field_names.PERCENTILE_FIELD_SUFFIX,
-    ]
-
-
-@pytest.fixture
-def expected_wildfire_percentile_columns():
-    return [
-        field_names.HIGH_FUTURE_WILDFIRE_RISK_FIELD,
-        field_names.FUTURE_WILDFIRE_RISK_FIELD
-        + field_names.PERCENTILE_FIELD_SUFFIX,
-    ]
-
-
-@pytest.fixture
-def score_col():
-    return field_names.SCORE_N_COMMUNITIES
-
-
-@pytest.fixture
-def score_col_with_donuts():
-    return field_names.FINAL_SCORE_N_BOOLEAN
-
-
-@pytest.fixture
-def lead_paint_columns():
-    """Returns lead paint percentile column, Median house value percentile column,
-    and thresholded boolean column for the proxy (in order).
-    Does not return thresholds
-    """
-    return [
-        field_names.LEAD_PAINT_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-        field_names.MEDIAN_HOUSE_VALUE_FIELD
-        + field_names.PERCENTILE_FIELD_SUFFIX,
-        field_names.LEAD_PAINT_PROXY_PCTILE_THRESHOLD,
-    ]
-
+@dataclass
+class PercentileTestConfig:
+    percentile_column_name: str
+    threshold_column_name: str
+    threshold: float
+    percentile_column_need_suffix: bool = True
 
-@pytest.fixture
-def lead_paint_proxy_home_value_threshold():
-    return ScoreNarwhal(final_score_df).MEDIAN_HOUSE_VALUE_THRESHOLD
+    @property
+    def full_percentile_column_name(self):
+        if self.percentile_column_need_suffix:
+            return (
+                self.percentile_column_name
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            )
+        return self.percentile_column_name
 
 
 ### TODO: we need to blow this out for all eight categories
-def _helper_test_percentile_against_threshold(
-    df, pctile_column, thresholded_column, threshold
-):
+def _check_percentile_against_threshold(df, config: PercentileTestConfig):
     """Note - for the purpose of testing, this fills with False"""
     minimum_flagged = (
```

Comment:
I did a poor job naming these variables. We might want them to be `minimum_flag`ok` or something to indicate that they are booleans?